src/components/MultiHopTrade/hooks/useGetTradeQuotes/useGetTradeQuotes.tsx

```diff
@@ -62,7 +63,10 @@ const getMixPanelDataFromApiQuotes = (quotes: ApiQuote[]): GetMixPanelDataFromAp
     })
     .filter(isSome)
 
-  return { quoteMeta, sellAssetId, buyAssetId, sellAmountUsd }
+  // Add a version string, in the form of an ISO 8601 standard basic format date, to the JSON blob to help with reporting
+  const version = '20230823'
```

Comment:
Just needed a unique string I could use to filter the data in Jupyter Notebook.

Reply:
i thought 8601 was delimited by `-`, e.g `2023-08-23` ?

Reply:
That's the "extended" version. The "normal" version is without the delimiters 😁